_includes/default_roles/crier.md

```diff
@@ -1,19 +1,24 @@
-The _Crier_ reminds people in _Slack_ of the upcoming meeting.
+A _Crier_ reminds us in Slack of upcoming meetings.
 
-**1 Business Day Before Meeting**
+**One Business Day Before The Meeting**
 
-* [ ] Announce the meeting in our [Slack][chat]{:target="_blank"}.
-* [ ] Use the `Share` arrow icon to share that announcement with the [#general] channel.
+* [ ] *Announce* the meeting in our [Slack][chat]{:target="_blank"}.
+* [ ] *Send the announcement* to [#general] using the `Share` arrow.
 
 <img src="https://user-images.githubusercontent.com/9609562/220438340-2fed944a-142b-4217-bcae-5c0e0110ed05.png" width="500px" />
 
+**Behaviors**
+
+* Recruit external members.
+  * Present to the InnerSource community.
+  * Invite your friends!
+
 **Tips**
 
-* Ensure that the announcement has enough information for people to join the meeting even if they don't have the calendar invite.
-  - Meeting day/time
-  - Call info
-* Try to make the announcement engaging so that even people that have not participated before may be interested in joining.
-A good way to do that is to incorporate interesting agenda items from the calendar invite into the announcement.
+*  Make the announcement clear enough for those without a calendar invite.
+     - Meeting day/time
+     - Call info
```

Comment:
Should we use the hyphen-formatting in the list further up in this file too?

Reply:
Not sure what this means?

Reply:
Here's the original text.

What's unclear?

```
 Ensure that the announcement has enough information for people to join the meeting even if they don't have the calendar invite.
  - Meeting day/time
  - Call info
```

Reply:
Documented https://github.com/InnerSourceCommons/working-group-roles/pull/4